src/gui/tray/ActivityListModel.cpp

```diff
@@ -444,13 +444,15 @@ void ActivityListModel::triggerDefaultAction(int activityIndex) const
 
         const auto baseName = QFileInfo(basePath).fileName();
 
-        ConflictDialog dialog;
-        dialog.setBaseFilename(baseName);
-        dialog.setLocalVersionFilename(conflictedPath);
-        dialog.setRemoteVersionFilename(basePath);
-        if (dialog.exec() == ConflictDialog::Accepted) {
+        auto dialog = new ConflictDialog;
+        dialog->setBaseFilename(baseName);
+        dialog->setLocalVersionFilename(conflictedPath);
+        dialog->setRemoteVersionFilename(basePath);
+        dialog->setAttribute(Qt::WA_DeleteOnClose);
+        connect(dialog, &ConflictDialog::accepted, [folder]() {
             folder->scheduleThisFolderSoon();
-        }
+        });
+        dialog->open();
```

Comment:
To be on the safe side I'd also force raising it and getting the focus. Since this dialog can't have a parent there's a slight chance it might "pop under" something else. You can use `ownCloudGui::raiseDialog()` for that.